src/constants/baseUrl.js

```diff
@@ -0,0 +1,4 @@
+const BASE_URL =
+  "https://cdn.jsdelivr.net/gh/fawazahmed0/currency-api@1/latest";
+
+export default BASE_URL;
```

Comment:
We don't want to default export on a constant because it would allow to rename it and we don't want this by definition of a constant 🗡️ 
```suggestion
export const BASE_URL =
  "https://cdn.jsdelivr.net/gh/fawazahmed0/currency-api@1/latest";
```

Reply:
changed :)